pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -26,6 +27,14 @@ mkDerivation rec {
     sha256 = "sha256-vfp/vacIItlPP5dR7jzDT7oOUNFnjvvdR46yi79EJKU=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix_ffmpeg_4.4.patch"; # to be removed with next release
```

Comment:
The comment is to be put before the patch definition, namely, two lines above.

But I will not complain on this now.